scripts/ubuntu-bartender/ubuntu-bartender

```diff
@@ -295,6 +313,12 @@ then
   exit 255
 fi
 
+if [ "$AWS_ARM_BUILD" = "true" ]
+then
+  AWS_INSTANCE_TYPE="m6g.large"
```

Comment:
Not blocking but we should check to see if --aws-instance-type passed and not override if it is set.

Reply:
should we do the same for the name filter as well? I actually had started to write those, and a piece of me sort of wants it all to be mutually exclusive